### DISCUSSION TITLE
991CN X教程第二版大纲征求意见稿

### DISCUSSION THREAD

**Discussion by Physics365** [Physics365/991CN-X-CW-Decompilation#46] [Ideas]
教程第二版大纲征求意见稿。

原来的那版我觉得架构不太合理。

手写的，待会可能会发电子版。

**有意见下面发（如调整章节顺序，缺失内容等）**

_标 * 的为选学内容（阅读时可以暂时跳过）_

![IMG_20240429_233717.jpg](https://github.com/Physics365/991CN-X-CW-Decompilation/assets/154725076/82224904-79ae-460a-ae47-b8ef6062d8ba)

![IMG_20240429_233724.jpg](https://github.com/Physics365/991CN-X-CW-Decompilation/assets/154725076/0321ef5d-c8b0-42d2-acaa-0a51a88526f4)

![IMG_20240429_233729.jpg](https://github.com/Physics365/991CN-X-CW-Decompilation/assets/154725076/8d2e6549-d973-45d2-adfd-3fbcfeffb991)

![IMG_20240429_233734.jpg](https://github.com/Physics365/991CN-X-CW-Decompilation/assets/154725076/8280f0e2-61e5-45a9-a488-ad5350b278eb)

@EwTE-H @Wenti-D @qiufuyu123

**Comment by Wenti-D** [comment]
什么是「反向面向编程」？(滑稽)

**Reply by Physics365** [reply]
返回导向编程

**Comment by Wenti-D** [comment]
这个大纲作为写作思路挺不错的，但章节划分不一定要这么来，视内容长短而优化，不然有可能出现「一节就一页，一页几小节」这样过于密集的划分。

**Reply by Physics365** [reply]
意思是整体顺序没问题咯

**Reply by Wenti-D** [reply]
虽然我个人觉得开头上汇编还是有点怪怪的，但如果后面必须要汇编来辅助讲解的话那我没意见。

**Reply by Physics365** [reply]
加了选学标签，可以先学，也可以后面用到了看。
（事实上马上就会用到）

**Reply by Physics365** [reply]
可以加深对处理器，内存等的理解
后面输入这一章的光标逻辑也可以用汇编写

**Comment by fish4terrisa-MSDSM** [comment]
附录是不是可以加一个所有单字节的字符的表格，加上比如说是否可以直接用lbf转换器刷出和其本身的功能（比如Intg(或者个别有特殊用途的@之类）的信息，可能会蛮有用的说（也就是说，扩展字符表）

**Reply by Physics365** [reply]
OK

**Comment by Physics365** [comment]
五一我物竞培训，没有电脑，你们或许可以先写一点

**Comment by qiufuyu123** [comment]
那么我们教程排版格式怎么说

**Reply by Physics365** [reply]
可以让@Wenti-D 搞一个模板的最终稿

**Reply by Wenti-D** [reply]
就我个人的经验而言，在还没写作时制定模板不太可能包罗万象面面俱到，有些想法、需求是在写作过程中产生的，到那时再思考或许也不迟，所以模板终稿不是一开始就能定好的。不过可以先照之前的模板写着？有想改的提一下意见就好。

以及五月上旬我会很忙，无暇顾他。

**Reply by Physics365** [reply]
OK

**Comment by Wenti-D** [comment]
突然想到了我以前写教程时的一些方法论，请按照以下几个问题审视我们的内容：
1. 我们的受众是谁？
2. 受众有哪些已有知识？
3. 受众有多少理解能力？
4. 我们所讲解的知识对于受众而言有无必要？
5. 对受众有必要的知识是否在其理解能力范围内？
6. 有必要而难以理解的知识如何清晰易懂地传授给受众？

**Reply by Physics365** [reply]
2，4我晚上在贴吧做个调查，其他我晚上下课想想

**Reply by Wenti-D** [reply]
啊，这些问题不用很快给出明确答复，而是在整个过程中都需要时时思考的。

**Reply by Physics365** [reply]
好的

**Comment by EwTE-H** [comment]
有动静吗

**Reply by fish4terrisa-MSDSM** [reply]
我大概6月中旬会有空，到时候可以参与一下

**Reply by ljm673** [reply]
终于有动静了😅不过我要等到6月十几号才有空😅

**Reply by EwTE-H** [reply]
但是已经过去两个月了

**Comment by Physics365** [comment]
最近有点忙，还在研究汇编，考试也多，可能我要等到五月底有空
